app/build.gradle.kts

```diff
@@ -0,0 +1,433 @@
+    }
+
+    create("staging") {
+      val stagingManifestEndpoint: String by project
+      val stagingFallbackApiEndpoint: String by project
+      val stagingMixpanelToken: String by project
+
+      dimension = "track"
+      applicationIdSuffix = ".staging"
+      versionNameSuffix = "-demo"
+      buildConfigField("String", "MANIFEST_ENDPOINT", "\"$stagingManifestEndpoint\"")
+      buildConfigField("String", "FALLBACK_ENDPOINT", "\"$stagingFallbackApiEndpoint\"")
+      buildConfigField("String", "MIXPANEL_TOKEN", "\"$stagingMixpanelToken\"")
+      buildConfigField("boolean", "DISABLE_SCREENSHOT", "false")
+      buildConfigField("boolean", "ALLOW_ROOTED_DEVICE", "true")
+    }
+
+    create("sandbox") {
+      val sandboxManifestEndpoint: String by project
+      val sandboxFallbackApiEndpoint: String by project
+      val sandboxMixpanelToken: String by project
+
+      dimension = "track"
+      applicationIdSuffix = ".sandbox"
+      versionNameSuffix = "-sandbox"
+      buildConfigField("String", "MANIFEST_ENDPOINT", "\"$sandboxManifestEndpoint\"")
+      buildConfigField("String", "FALLBACK_ENDPOINT", "\"$sandboxFallbackApiEndpoint\"")
+      buildConfigField("String", "MIXPANEL_TOKEN", "\"$sandboxMixpanelToken\"")
+      buildConfigField("boolean", "DISABLE_SCREENSHOT", "false")
+      buildConfigField("boolean", "ALLOW_ROOTED_DEVICE", "true")
+    }
+
+    create("security") {
+      val securityManifestEndpoint: String by project
+      val securityFallbackApiEndpoint: String by project
+      val securityMixpanelToken: String by project
+
+      dimension = "track"
+      applicationIdSuffix = ".security"
+      versionNameSuffix = "-security"
+      buildConfigField("String", "MANIFEST_ENDPOINT", "\"$securityManifestEndpoint\"")
+      buildConfigField("String", "FALLBACK_ENDPOINT", "\"$securityFallbackApiEndpoint\"")
+      buildConfigField("String", "MIXPANEL_TOKEN", "\"$securityMixpanelToken\"")
+      buildConfigField("boolean", "DISABLE_SCREENSHOT", "true")
+      buildConfigField("boolean", "ALLOW_ROOTED_DEVICE", "false")
+    }
+
+    create("production") {
+      val productionManifestEndpoint: String by project
+      val productionFallbackApiEndpoint: String by project
+      val productionMixpanelToken: String by project
+
+      dimension = "track"
+      buildConfigField("String", "MANIFEST_ENDPOINT", "\"$productionManifestEndpoint\"")
+      buildConfigField("String", "FALLBACK_ENDPOINT", "\"$productionFallbackApiEndpoint\"")
+      buildConfigField("String", "MIXPANEL_TOKEN", "\"$productionMixpanelToken\"")
+      buildConfigField("boolean", "DISABLE_SCREENSHOT", "true")
+      buildConfigField("boolean", "ALLOW_ROOTED_DEVICE", "false")
+    }
+  }
+
+  variantFilter = Action<VariantFilter> {
+    if (name == "qaRelease" ||
+        name == "stagingDebug" ||
+        name == "sandboxDebug" ||
+        name == "productionDebug" ||
+        name == "securityDebug"
+    ) {
```

Comment:
```
val variantsToIgnore = setOf(/* */)
if (name in variantsToIgnore) {
  ignore = true
}
```

Might be slightly cleaner to do it this way?

Reply:
Ah, good point.